home_5/src/dijkstra.spec.ts

```diff
@@ -0,0 +1,80 @@
+// @ts-nocheck
+
+import { Dijkstra } from "./dijkstra";
+import { Edge } from "./edge";
+import { Graph } from "./graph";
+import { Vertex } from "./vertex";
+
+describe("PriorityQueue", () => {
+  let vertices = [];
+  let edges = [];
+  let graph: Graph;
+  let vertex1: Vertex;
+  let vertex2: Vertex;
+  let vertex3: Vertex;
+  let vertex4: Vertex;
+  let vertex5: Vertex;
+
+  beforeEach(() => {
+    vertex1 = new Vertex("1");
```

Comment:
tried another graph and didn't get right results for it:
<img width="1299" alt="image" src="https://user-images.githubusercontent.com/145040/233212236-6cc14df1-175a-41b7-b4d4-554b851a7b3e.png">